mappings/net/minecraft/world/IWorld.mapping

```diff
@@ -15,5 +15,7 @@ CLASS bbq net/minecraft/world/IWorld
 	METHOD g getLevelProperties ()Lcih;
 	METHOD g getLocalDifficulty (Let;)Lafn;
 	METHOD i getSaveHandler ()Lcii;
+	METHOD k getSkyAngle (F)F
```

Comment:
There was some discussion on this. It could be "pitch" instead of "angle". We could also use "celestial" instead of "sky".

Reply:
[0, 1), used to rotate the sky and the clock item among other things.

Reply:
If it,s [0, 1), is it really an angle? Does it map like a scaled angle or like a height?

Reply:
Turns is a valid angle unit, as is radians and degrees.

Reply:
Fair.